stock_picking_purchase_order_alert/models/stock_picking_type.py

```diff
@@ -0,0 +1,24 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    display_quantity_alert_percentage = fields.Boolean(
+        string="Display Quantity Alert Percentage",
```

Comment:
The "percentage" word appear too many times in the view IMO
```suggestion
        string="Display Quantity Alert",
```